protobuf/as/external/api/device.proto

```diff
@@ -136,6 +136,15 @@ service DeviceService {
             get: "/api/devices/{dev_eui}/events"
         };
     }
+
+    // ClearDeviceDevNonce deletes the device older activation records for the given DevEUI.
+    //   * These are clear older DevNonce records from device activation records in Network Server
+    //   * These clears all DevNonce records but keeps latest 20 records for maintain device activation status
+    rpc ClearDeviceDevNonce(ClearDevNonceRequest) returns (google.protobuf.Empty) {
```

Comment:
Hi @sagar-patel-sls, sorry for the late review. I have been and I'm spending a lot of time on the AS + NS merge as I want to get this out asap.

I think I would call this `ClearDeviceNonces` / `ClearDeviceNoncesRequest` as it clears multiple items, not a single entry.

Other than that, it looks good to me. Could you make the other PRs before I merge this? Then I can merge this one, tag a new release and push this as a change to the other PRs.

Reply:
Hi @brocaar 

i have made changes as per your comments.